.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
+
+on: push
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: macOS-10.15
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Used to host cibuildwheel
+      - uses: actions/setup-python@v2
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==1.9.0
+
+      - name: Install sundials
+        run: |
+          brew update
+          brew install sundials
```

Comment:
why do we still need to brew install sundials if we now have the wheel?

Reply:
That's because this workflow is building the wheel. In order to compile the idaklu module (written in C++), this workflow (or anyone building pybamm from source) must have sundials installed. The resulting (compiled/binary) module still depends on sundials, but the required sundials libraries are bundled into the wheel, meaning that anyone with the wheel doesn't need to install sundials. But to build the wheel, it is needed.